test/adapters/test_mfem.cpp

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: 2022 Dennis Gläser <dennis.glaeser@iws.uni-stuttgart.de>
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "mfem.hpp"
+
+#include <gridformat/traits/mfem.hpp>
+#include <gridformat/vtk/vtu_writer.hpp>
+#include <gridformat/vtk/vtp_writer.hpp>
+
+#include "../make_test_data.hpp"
+#include "../testing.hpp"
+
+
+std::array<double, 2> cell_center(const mfem::Mesh& mesh, const std::integral auto elem_idx) {
+    const auto elem_ptr = mesh.GetElement(elem_idx);
```

Comment:
could use mesh.GetElementCenter(idx, vector)

Reply:
ah cool, will change it

Reply:
Interestingly, this interface seems to require a non-const `Mesh`: https://github.com/mfem/mfem/blob/v4.5.2/mesh/mesh.hpp#L1832

I noticed because in the test I was constructing a `const mesh = ...`. It's a bit unintuitive to me why such function should require non-const access.